goth_tests/pyproject.toml

```diff
@@ -30,7 +30,7 @@ pytest-asyncio = "0.21"
 pytest-split = "^0.8.1"
 goth = "0.15.8"
 # to use development goth version uncomment below
-# goth =  { git = "https://github.com/golemfactory/goth.git", rev = "0845cad9a4d590b06849c6888db7f060788eb7d7" }
+# goth =  { git = "https://github.com/golemfactory/goth.git", rev = "47df5fa77a4e04239cf0f48d9d9a0b7e3c7468e4" }
```

Comment:
This does nothing, the line is commented out and the version of goth in `poetry.lock` is still the same.